src/client/src/rt-platforms/platformAdapter.ts

```diff
@@ -1,89 +0,0 @@
-import { PlatformFeatures, PlatformType, WindowConfig } from './types'
```

Comment:
completely removed this in favor of `Platfrom` interface in `platfrom.ts` and `getPlatfromAsync.ts`